src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1458,8 +1458,6 @@ static void _cloneNode(SvgNode* from, SvgNode* parent)
     for (uint32_t i = 0; i < from->child.cnt; ++i, ++child) {
         _cloneNode(*child, newNode);
     }
-
-    _freeNode(newNode);
```

Comment:
Is there no case of null parent?
=> if (!parent) _freeNode(newNode);

Reply:
It is not necessary. Becuase if parent is null, a leak occurs because the created node is not connected. so. If parent is null, it should be returned at line 1452. However, _cloneNode is only used in UseNode case. And parent is null means that createNode fails, so it is not necessary for the current logic.

Reply:
then please make it return at 1452 
if (!from || !parent) return;